spec/index.bs

```diff
@@ -3313,22 +3334,41 @@ dictionary GPUComputePipelineDescriptor : GPUPipelineDescriptorBase {
                 1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
                 1. Create a new [=invalid=] {{GPUComputePipeline}} and return the result.
         </div>
-</dl>
 
-### {{GPUDevice/createReadyComputePipeline()}} ### {#device-createReadyComputePipeline}
+    : <dfn>createReadyComputePipeline(descriptor)</dfn>
+    ::
+        Creates a {{GPUComputePipeline}}. The returned {{Promise}} resolves when the created pipeline
+        is ready to be used without additional delay.
 
-Same as {{GPUDevice/createComputePipeline()}}, but returns its result as a
-promise which doesn't resolve until the pipeline is ready to be used without
-additional delay.
+        If pipeline creation fails, the returned {{Promise}} resolves to an [=invalid=]
```

Comment:
Shouldn't it reject instead?

Reply:
This is how I wrote it in #871, but I don't remember the discussion that led to that. Maybe it should.

Reply:
I think it was because I thought it didn't make sense to send errors out twice: once in rejection and once in the error scope. I'm still not sure what's best.

I guess arguably any validation errors could result in rejection (programmer error), while backend pipeline creation failure could result in an invalid pipeline....

Reply:
I thought we had decided that errors in createReadyPipeline wouldn't be surfaced in the error scope and only through rejection.

Reply:
An argument for having them go to the error scope is that it preserves the completeness of "onuncapturederror" for telemetry usecases.

This is something I never resolved in ErrorConventions.md (it says "If the operation returns a Promise, it rejects (and maybe produces an error log entry).").

Reply:
Filed https://github.com/gpuweb/gpuweb/issues/1049